packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -396,6 +396,44 @@ export class WorkflowsController {
 		return true;
 	}
 
+	@Post('/:workflowId/archive')
+	@ProjectScope('workflow:delete')
+	async archive(req: WorkflowRequest.Delete) {
```

Comment:
Can we use a DTO instead? Same below.

Reply:
🤔 I can add separate types `WorkflowRequest.Archive` and `WorkflowRequest.Unarchive`, but they're just going to be aliases of the `type Get = AuthenticatedRequest<{ workflowId: string }>;` like the Delete is currently, there's no body to validate with DTOs?

Reply:
We have `@Param` for route parameters, see [example](https://github.com/n8n-io/n8n/blob/e6381e09e37dd40f7ba3222ec70b050fc08e0b06/packages/cli/src/controllers/api-keys.controller.ts#L80).